doc/modules/mod_inbox.md

```diff
@@ -5,18 +5,53 @@ To use it, enable mod\_inbox in the config file.
 
 ### Options
 
-* **backend** (atom, default: `rdbms`) - Database backend to use. For now, only `rdbms` is supported.
-* **reset_markers** (list, default: `[displayed]`) - List of atom chat markers that when sent, will reset the unread message counter for a conversation.
+#### `modules.mod_inbox.backend`
+* **Syntax:** string
+* **Default:** `"rdbms"`
+* **Example:** `backend = "rdbms"`
+
+Database backend to use. For now, only `rdbms` is supported.
+
+#### `modules.mod_inbox.reset_markers`
+* **Syntax:** array of strings, out of `"displayed"`, `"received"`, `"acknowledged"`
+* **Default:** `["displayed"]`
+* **Example:** `reset_markers = ["received"]`
+
+List of chat markers that when sent, will reset the unread message counter for a conversation.
 This works when [Chat Markers](https://xmpp.org/extensions/xep-0333.html) are enabled on the client side.
-Possible values are from the set: `displayed`, `received`, `acknowledged`. Setting as empty list (not recommended) means that no chat marker can decrease the counter value.
-* **groupchat** (list, default: `[muclight]`) - The list indicating which groupchats will be included in inbox.
+Setting as empty list (not recommended) means that no chat marker can decrease the counter value.
```

Comment:
Is it possible to set an option to an empty list in TOML?

Reply:
Yes it is (technically an empty array).